x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/dataframe/evaluation/outlierdetection/OutlierDetection.java

```diff
@@ -50,30 +51,33 @@ public static OutlierDetection fromXContent(XContentParser parser) {
         return PARSER.apply(parser, null);
     }
 
-    static QueryBuilder actualIsTrueQuery(String actualField) {
+    public static QueryBuilder actualIsTrueQuery(String actualField) {
         return QueryBuilders.queryStringQuery(actualField + ": (1 OR true)");
     }
 
     /**
-     * The field where the actual class is marked up.
-     * The value of this field is assumed to either be 1 or 0, or true or false.
+     * The collection of fields in the index being evaluated.
+     *   fields.getActualField() is assumed to either be 1 or 0, or true or false.
+     *   fields.getPredictedField() is assumed to be a number in [0.0, 1.0].
```

Comment:
> actual field is a number or bool?

It can either be a number of bool (it was like this from the beginning of `Evaluate API`). 
Here is the query that select positive docs:
```
    public static QueryBuilder actualIsTrueQuery(String actualField) {
        return QueryBuilders.queryStringQuery(actualField + ": (1 OR true)");
    }
```

> predicted field is a number?

There was a mistake in this comment. I've just fixed it.